apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/clear_ccw_pipeline_load.sql

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
Created a runbook that explains how to use this script for re-running a pipeline load: https://github.com/CMSgov/beneficiary-fhir-data/wiki/How-to-clean%E2%80%90up-and-reload-test-data-from-a-CCW-pipeline-load